projects/Draupnir/default.nix

```diff
@@ -32,5 +32,10 @@
         tests.basic = "${sources.inputs.nixpkgs}/nixos/tests/matrix/draupnir.nix";
       };
     };
+    demo.vm = {
+      module = ./example.nix;
+      description = "Deployment for demo purposes";
```

Comment:
Is there a specific workflow for testing draupnir?

**PS:** If there isn't, this project is actually a good candidate for https://github.com/ngi-nix/ngipkgs/issues/1139

Reply:
There isn't one at the moment and the only tests are the one in nixpkgs.  Maybe we shouldn't expose the demo vm.

Reply:
> Maybe we shouldn't expose the demo vm.

Yes, but I believe we can still add one and keep refining it. That's the idea behind https://github.com/ngi-nix/ngipkgs/issues/1139.